pyaptly/conftest.py

```diff
@@ -102,13 +108,16 @@ def test_snapshot_create_basic(environment, config, freeze):
 
 @pytest.fixture()
 def config(request):
-    """Get a config.
+    """
+    Get a configuration for testing.
 
-    Can be configured with:
+    This fixture can be configured with a specific configuration file. For
+    example, to use "mirror-extra.toml" as the configuration, annotate a test
+    function like this:
 
     ```python
-    @pytest.mark.parametrize("config", ["mirror-extra.toml"], indirect=True)
-    def test_mirror_create(environment, config, caplog):
+    @pytest.mark.parametrize("freeze", ["2012-10-10 10:10:10"], indirect=True)
+    def test_snapshot_create_basic(environment, config, freeze):
```

Comment:
```suggestion
    def test_mirror_create(environment, config, caplog):
```

Reply:
Sorry. I had to fix that 100 times. And the last time I forgot!